Tests/scripts/update_conf_json.py

```diff
@@ -0,0 +1,112 @@
+import os
+import yaml
+import json
+from distutils.version import LooseVersion
+
+PACKS_DIR = './Packs'
+CONF_JSON_PATH = './Tests/conf.json'
+INTEGRATIONS_DIR = 'Integrations'
+TEST_PLAYBOOKS_DIR = 'TestPlaybooks'
+
+
+def get_integration_data(file_path):
+    with open(file_path) as data_file:
+        yaml_file = yaml.safe_load(data_file)
+        return yaml_file['commonfields']['id'], yaml_file.get('fromversion', '0.0.0')
+
+
+def get_playbook_id(file_path):
+    with open(file_path) as data_file:
+        yaml_file = yaml.safe_load(data_file)
+        return yaml_file['id']
+
+
+def get_fromversion(integrations):
+    max_fromversion = '0.0.0'
+    for integration_id, fromversion in integrations:
+        if LooseVersion(fromversion) > LooseVersion(max_fromversion):
+            max_fromversion = fromversion
+
+    return max_fromversion
+
+
+def calc_conf_json_object(integrations, test_playbooks):
+    conf_objects = [{'playbookID': test_playbook} for test_playbook in test_playbooks]
+    fromverion = get_fromversion(integrations)
```

Comment:
I think we also have `fromversion` for the test playbook, not sure if still relevant (we had an integration we two test playbook that differs in a few tasks that appeared only for a specific version)